root/label/LabelIndex.js

```diff
@@ -51,6 +59,20 @@ const LabelIndex = ({
       entity={label}
       numberOfRevisions={numberOfRevisions}
     />
+    {renamedFrom.length ? (
+      <RelatedEntitiesDisplay title={l('Renamed from')}>
+        {commaOnlyList(renamedFrom.map(
+          label => <DescriptiveLink entity={label} key={label.gid} />,
+        ))}
+      </RelatedEntitiesDisplay>
+    ) : null}
+    {renamedInto.length ? (
+      <RelatedEntitiesDisplay title={l('Renamed into')}>
```

Comment:
This doesn't seem wrong but "Renamed to" sounds a bit more common.
```suggestion
      <RelatedEntitiesDisplay title={l('Renamed to')}>
```